src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj

```diff
@@ -12,7 +12,7 @@
     <OutDir>$(OutDir)lib\</OutDir>
     <IntDir>$(Configuration)\lib\</IntDir>
     <ConfigurationType>StaticLibrary</ConfigurationType>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```

Comment:
Yet another "off by 0.0.629.0" error... so typical!

Reply:
There are two problems in programming...